src/CircularLoader/index.js

```diff
@@ -5,7 +5,7 @@ import cx from 'classnames'
 
 import styles from './styles.js'
 
-const CircularProgress = ({ small, large, className }) => (
+const CircularLoader = ({ small, large, className }) => (
```

Comment:
Renamed to match the terms in https://github.com/dhis2/design-system/blob/master/atoms/loading.md

#237